site/content/tutorial/01-introduction/05-nested-components/text.md

```diff
@@ -2,17 +2,21 @@
 title: Nested components
 ---
 
-It would be impractical to put your entire app in a single component. Instead, we can import components from other files and include them as though we were including elements.
+It would be impractical to put your entire app in a single component. Instead, we can import components from other files and then use them as though we were including elements.
 
-Add a `<script>` tag that imports `Nested.svelte`...
+NOTE: Remember that in Svelte, an application is composed from one or more components and each component is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together, written into a `.svelte` file.
+
+We now present you 2 files in the editor on the right (upper bar) to click on, `App.svelte` and `Nested.svelte`.
```

Comment:
I think this can be shortened to:

```suggestion
Each component is a reusable self-contained block of code that encapsulates HTML, CSS, and JavaScript that belong together, written into a `.svelte` file.
```

without losing much detail.